[2023-04-25T13:02:24.795+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-04-25T13:02:23.362915+00:00 [queued]>
[2023-04-25T13:02:24.803+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-04-25T13:02:23.362915+00:00 [queued]>
[2023-04-25T13:02:24.805+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-25T13:02:24.807+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-04-25T13:02:24.808+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-25T13:02:24.829+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-04-25 13:02:23.362915+00:00
[2023-04-25T13:02:24.851+0000] {standard_task_runner.py:55} INFO - Started process 12477 to run task
[2023-04-25T13:02:24.873+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-04-25T13:02:23.362915+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmplm0lscdj']
[2023-04-25T13:02:24.874+0000] {standard_task_runner.py:83} INFO - Job 41: Subtask is_api_available
[2023-04-25T13:02:25.041+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-04-25T13:02:23.362915+00:00 [running]> on host bebb68623a17
[2023-04-25T13:02:25.207+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-04-25T13:02:23.362915+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-25T13:02:23.362915+00:00
[2023-04-25T13:02:25.212+0000] {http.py:120} INFO - Poking: api/
[2023-04-25T13:02:25.240+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-04-25T13:02:25.250+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-04-25T13:02:25.755+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-04-25T13:02:25.783+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20230425T130223, start_date=20230425T130224, end_date=20230425T130225
[2023-04-25T13:02:25.859+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-25T13:02:25.888+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
